packages/datasources/src/lib/types.ts

```diff
@@ -95,9 +95,10 @@ export type ContractConfig =
     };
 
 /**
- * DatasourceMap encodes the set of known Datasources within an ENS namespace, keyed by DatasourceName.
+ * ENSNamespace encodes a set of known Datasources, together constituting an ENS namespace.
```

Comment:
```suggestion
 * ENSNamespace encodes a set of known Datasources associated with the same ENS namespace.
```

This is a key distinction.